assets/js/cookie.js

```diff
@@ -60,8 +62,16 @@ function cookieBanner() {
 		} else {
 			document.body.classList.replace("dark-theme", "light-theme");
 			setCookie("theme", "light", 60, true);
-		}
+		};
 	});
 
-	setTimeout(() => fadeIn(document.getElementById('cookie-container')), 1000);
+	document.getElementById("lang-toggle").addEventListener("change", function() {
+		if (document.documentElement.lang.includes("en")) {
+			reloadText("de");
+		} else {
+			reloadText("en");
+		};
+	});
+	if (reloadText) reloadText(getLanguage());
+	if (getLanguage() != "de") document.getElementById("lang-toggle").checked = true
 };
```

Comment:
This seems a bit strange to have in the cookie.js file (same for the theme toggle), maybe add another file for this (e.g. named "script.js")?